src/Microsoft.IdentityModel.Validators/AadTokenValidationParametersExtension.cs

```diff
@@ -41,6 +43,13 @@ public static void EnableAadSigningKeyIssuerValidation(this TokenValidationParam
             };
         }
 
+        internal const string DontFailOnMissingTidSwitch = "Switch.Microsoft.IdentityModel.DontFailOnMissingTidValidateIssuerSigning";
+
+        private static bool DontFailOnMissingTid()
```

Comment:
If this extension represents the correct way to validate a token from AAD, i don't think we should have an opt out.